transforms/src/__testfixtures__/button-v0-to-v1.output.tsx

```diff
@@ -0,0 +1,42 @@
+import { ButtonV1 as Button } from '@fluentui/react-native';
```

Comment:
This file is generated by the tool? Or is it the handcrafted test case that we use to compare against?

Reply:
It's a handcrafted test case that we use to compare against.

Reply:
Specifically it's a copy of https://github.com/microsoft/fluentui-react-native/blob/main/apps/fluent-tester/src/TestComponents/Button/deprecated/ButtonFocusTest.tsx